kratos/python_scripts/point_output_process.py

```diff
@@ -181,8 +170,12 @@ def __SearchPoint(self):
             my_rank = comm.Rank()
         writing_rank = comm.MaxAll(my_rank) # The partition with the larger rank writes
 
-        if my_rank == writing_rank:
+        if my_rank == -1:
```

Comment:
I think this is a bit confusing to name this variable like that. I would assume at first glance that this indicates my rank, not if results were found in my rank.

Rather than that its ok.